extensions/wikia/UserProfilePageV3/UserIdentityBox.class.php

```diff
@@ -314,7 +317,10 @@ public function saveUserData($data) {
 						}
 					}
 
-					if ($option === 'gender') {
+					if ($option === 'hideEditsWikis') {
+						// hideEditsWikis is a preference, everything else is an attribute
+						$this->user->setGlobalPreference($option, $data->$option);
+					} elseif ($option === 'gender') {
 						$this->user->setGlobalAttribute(self::USER_PROPERTIES_PREFIX . $option, $data->$option);
```

Comment:
I'm not sure how it works, but here is the line where you add USER_PROPERTIES_PREFIX to the 'gender' key, and in the line #216 it is filtered out, and the prefix is not added. Is this correct?